update readme (#5)
giantswarm/alloy-app: README.md

**What is this app?**

Alloy is an OpenTelemetry collector with support for metrics, logs, traces, and profiles.

More details at https://github.com/grafana/alloy

**Why did we add it?**

> We added Alloy in order to be able to improve our Observability stack and provides additional capabilities towards observability data ingestion and transformation.

Reviewer: ```suggestion
We added Alloy in order to be able to improve our Observability platform and provide additional capabilities towards observability data ingestion and transformation.
```

> We added Alloy in order to be able to improve our Observability stack and provides additional capabilities towards obseravability data ingestion and transformation.

Reviewer: ```suggestion
We added Alloy in order to be able to improve our Observability stack and provides additional capabilities towards observability data ingestion and transformation.
```

**Who can use it?**

> Anyone with a need to collect obseravability data and who need an OpenTelemetry compatible collector.

Reviewer: ```suggestion
Anyone with a need to collect observability data and who need an OpenTelemetry compatible collector.
```
